2_gyum/aladin.py

```
@@ -0,0 +1,168 @@
+# - requests.get(url): 지정한 URL에 HTTP GET 요청을 보냅니다.
+# - BeautifulSoup(html, 'html.parser'): 응답 받은 HTML을 BeautifulSoup을 사용해 파싱합니다.
+
+# In[40]:
+
+
+from bs4 import BeautifulSoup
+import requests
+
+# 알라딘 베스트셀러 페이지 URL
+url = "https://www.aladin.co.kr/shop/common/wbest.aspx?BestType=Bestseller&BranchType=1&CID=0&page=1&cnt=1000&SortOrder=1"
+response = requests.get(url)  # 요청 보내기
```

Ruff (0.13.1)

Probable use of `requests` call without timeout

(S113)